lib/features/home/home_viewmodel.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: public_member_api_docs
```

Comment:
Add this `public_member_api_docs: false` instead to the `analysis_options.yaml` file.